src/Diagnostics.DataProviders/DataProviders/Base/SupportObserverDataProviderBase.cs

```diff
@@ -63,16 +63,38 @@ public Task<dynamic> GetResource(string resourceUrl)
                 throw new FormatException(exceptionMessage, ex);
             }
 
-            if (uri.Host.Contains(allowedHosts[0]) || uri.Host.Contains(allowedHosts[1]) || Configuration.ObserverLocalHostEnabled)
+            if (uri.Host.Contains(allowedHosts[0]) || uri.Host.Contains(allowedHosts[1]))
             {
                 return GetWawsObserverResourceAsync(uri);
```

Comment:
So it looks like even without checking if `Configuration.ObserverLocalHostEnabled`, the logic will still send the request to LocalHostObserver in `SendObserverRequestAsync` because we strip the base URL in `GetWawsObserverResourceAsync` and append the URL path to the HttpClient base URI. Did you remove this in the if clause because it was redundant or some other reason?

Reply:
If the observer url is requested (`allowedHosts[0 or 1]`) then we can `GetWawsObserverResource` no matter what.
Else, we have a support-bay-api route, some of which can't hit the selfhost Observer. I check if I can convert the route; if yes, convert and send to Observer. If not, send back to support-bay-api